### PR TITLE
add linter, apply fixes

### PR DESCRIPTION
**Pull request by modernserf**
- Add eslint + standardJS
- apply linter fixes (fix whitespace, unused vars, some bugs, see line comments)

### CODE REVIEW

File: lib/httpServer.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
-const http = require('http')
 const util = require('util')
-const path = require('path')
 
 const Koa = require('koa')
 const body = require('koa-body')
 const cors = require('@koa/cors')
 const route = require('koa-route')
-const static = require('koa-static')
+const static_ = require('koa-static')
```
Comment (modernserf) [inline lib/httpServer.js]: `static` is a reserved word in JS, though I guess node will allow this somehow?

File: lib/socketServer.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
-const socket = require('koa-socket')
-const io = new socket()
+const Socket = require('koa-socket')
```
Comment (modernserf) [inline lib/socketServer.js]: `Socket` is a constructor and needs to be uppercase

File: lib/socketServer.js [hunk 3]
```diff
@@ -44,30 +44,30 @@ io.on('select', async context => {
...
+  // TODO: fire this on assert/retract, not by polling
+  const loop = () => {
+    if (queries) {
+      io.state.client.select(queries).doAll((res) => {
+        socket.emit('subscriptionFacts', res)
+      })
     }
 
-    loop()
+    timeout = setTimeout(loop, 10)
```
Comment (modernserf) [inline lib/socketServer.js]: no `let` here -- this is a reassignment

**Comment by jedahan** [comment]
Do we need eslint if we already have standard? Cant we just have `script.lint` and `script.lint:fix` just call standard directly?

File: package.json [hunk 4]
```diff
@@ -19,14 +26,17 @@
...
     "koa-static": "^4.0.2",
     "minimist": "^1.2.0",
+  },
+  "devDependencies": {
+    "eslint": "^4.16.0",
+    "eslint-config-standard": "^11.0.0-beta.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-standard": "^3.0.1"
```
Comment (modernserf) [inline package.json]: this needs to get fixed

Comment (jedahan) [reply package.json]: whoops good catch

**Comment by jedahan** [comment]
I'm gonna switch to just using standardjs, hopefully we won't need to modify rules via eslint

**Pull request merged**